examples/nidigital/configure-voltage-levels-and-termination-voltage.py

```diff
@@ -0,0 +1,197 @@
+
+     configure_voltage_response = nidigital_client.ConfigureVoltageLevels(nidigital_types.ConfigureVoltageLevelsRequest(vi = vi,
+         channel_list = channelList,
+         vil = vil,
+         vih = vih,
+         vol = vol,
+         voh = voh,
+         vterm = vterm))
+     CheckForError(vi, configure_voltage_response.status)
+     
+     while True:
+        if termSelect == 'h':
+            configure_termination_Mode_response = nidigital_client.ConfigureTerminationMode(nidigital_types.ConfigureTerminationModeRequest(vi = vi,
+                channel_list = channelList,
+                mode = nidigital_types.TerminationMode.TERMINATION_MODE_NIDIGITAL_VAL_HIGH_Z))
+            CheckForError(vi, configure_termination_Mode_response.status)
+
+        if termSelect == 'a':
+            configure_termination_Mode_response = nidigital_client.ConfigureTerminationMode(nidigital_types.ConfigureTerminationModeRequest(vi = vi,
+                channel_list = channelList,
+                mode = nidigital_types.TerminationMode.TERMINATION_MODE_NIDIGITAL_VAL_ACTIVE_LOAD))
+            CheckForError(vi, configure_termination_Mode_response.status)
+
+            
+            configure_active_load_levels_response = nidigital_client.ConfigureActiveLoadLevels(nidigital_types.ConfigureActiveLoadLevelsRequest(vi = vi,
+                channel_list = channelList,
+                iol = iol,
+                ioh = ioh,
+                vcom = vcom))
+            CheckForError(vi, configure_active_load_levels_response.status)
+
+        if termSelect == 't':
+            configure_termination_mode_response = nidigital_client.ConfigureTerminationMode(nidigital_types.ConfigureTerminationModeRequest(vi = vi,
+                channel_list = channelList,
+                mode = nidigital_types.TerminationMode.TERMINATION_MODE_NIDIGITAL_VAL_VTERM))
+            CheckForError(vi, configure_termination_mode_response.status)
+
+        if(termSelect == 'h' or termSelect == 'a' or termSelect == 't'):
+            break
+
+     burst_pattern_response = nidigital_client.BurstPattern(nidigital_types.BurstPatternRequest(vi = vi,
+        site_list = "",
+        start_label = "new_pattern",
+        select_digital_function = True,
+        wait_until_done = True,
+        timeout = 10))
+     CheckForError(vi, burst_pattern_response.status)
+
+     select_function_response = nidigital_client.SelectFunction(nidigital_types.SelectFunctionRequest(vi = vi,
+        channel_list = channelList,
+        function_raw = nidigital_types.SelectedFunction.SELECTED_FUNCTION_NIDIGITAL_VAL_DISCONNECT))
+     CheckForError(vi, select_function_response.status)
+
+     print('Done without error.\n')
+
+# If NI-Digital API throws an exception, print the error message  
+except grpc.RpcError as rpc_error:
+    error_message = rpc_error.details()
+    if rpc_error.code() == grpc.StatusCode.UNAVAILABLE :
+        error_message = f"Failed to connect to server on {server_address}"
```

Comment:
@harsha-bhushan 
When we give the server address we should also append the port number which will be useful. Something like "Failed to connect to server on localhost:31763". 
This is not specific to this example. We have implemented the same for other examples as well. So we could take this later if we plan to address this.

Reply:
Yes I agree. That was indeed the initial behavior. But it was changed to this late during the release as part of making examples uniform. While working on last MI driver example(Fgen) or after, we can fix this (for all other drivers) & anything else that's needed to maintain uniformity.

Reply:
@harsha-bhushan Resolving this comment for now, as no action is needed now!